pb/mgmt_pb/mgmt.proto

```diff
@@ -126,6 +140,14 @@ message GetTerminatorResponse {
     Terminator terminator = 1;
 }
 
+message SetTerminatorCostRequest {
+    string terminatorId = 1;
+    TerminatorPrecedence precedence = 2;
+    uint32 staticCost = 3;
+    uint32 dynamicCost = 4;
+    int32 updateMask = 5;
```

Comment:
I could have three bool fields instead of one mask. Mask just seemed a bit tidier. In the end, it's very unlikely that anyone will ever see this b/c the mask will get set by the CLI on the other side.  If you feel strongly about it, I can change it

Reply:
I don't feel _strongly_ about it, or I wouldn't have approved the PR. :-)

I think I generally prefer the more obvious solution, even if it's more verbose. Separate bools seems like a simpler thing to read about in code versus looking at bit masks. But it's pretty straightforward in this case, so it's not a big deal.

Reply:
This code will also likely be replaced relatively soon by our consolidated REST based management, which is the other reason I'm inclined to leave it :)

Reply:
Indeed. 💯